runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZonePlugin.java

```diff
@@ -135,6 +135,11 @@ public void onGameTick(GameTick event)
 				resetPointsPerHour();
 			}
 
+			if (overloadNotificationSend)
+			{
+				overloadNotificationSend = false;
+			}
```

Comment:
```suggestion
			overloadNotificationSend = false;
```

Reply:
I was thinking of doing it this way, but decided to imitate how `absorptionNotificationSend` is handled. 
```
if (!absorptionNotificationSend)
{
	absorptionNotificationSend = true;
}
```

Should we change that block of code to just `absorptionNotificationSend = true` along with `overloadNotificationSend = false`?

Reply:
I'm not really sure why it was written as it is, but leave it be. This, however, should not be written the same way since it makes no sense.

Reply:
I've also noticed `absorptionNotificationSend = true` is missing from startUp()